tests/test_openjdk.py

```diff
@@ -156,3 +174,56 @@ def test_jdk_extended(
 
     for check in params.expected_err_strings:
         assert check in testout.stderr
+
+
+@pytest.mark.parametrize(
+    "container_per_test",
+    CONTAINER_IMAGES_CASSANDRA,
+    indirect=["container_per_test"],
+)
+def test_jdk_cassandra(container_per_test):
+    """Starts the Cassandra DB and executes some write and read tests
+    using the cassandra-stress
+    """
+
+    cassandra_version = container_per_test.connection.check_output(
+        'git ls-remote --tags https://gitbox.apache.org/repos/asf/cassandra.git | grep -v alpha | grep -v beta | grep -v "{}" | tail -n 1 | cut -d "-" -f2'
```

Comment:
nitpick: if it is simpler to perform the parsing in python, then I'd suggest to use that instead, I find that usually much more obvious.
I would nevertheless suggest to add a simple sanity check, e.g. that you get only one version and/or that it matches a simple regex or something along those lines. It will make debugging of issues like `curl: failed to fetch https://downloads.apache.org/cassandra/remote hung up/apache-cassandra-remote hung up-bin.tar.gz 404` much easier :wink: